main.html

```diff
@@ -3,20 +3,76 @@
 <html>
 	<head>
 		<meta charset="UTF-8">
-		<title>pokedex</title>
+		<title>Pokedex</title>
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		<!-- make sure to link to your css file -->
+		<link rel="stylesheet" type="text/css" href="main.css">
 	</head>
 
 	<body>
 		<!-- include a header tag -->
-
+		<header>
+			<nav>
+				<a href="main.html"><img id="hamburger" src="assets/icons/hamburger_icon.png" alt="Hamburger Icon"></a>
+			</nav>
+			<h1>Pokedex</h1>
+			<img src="assets/icons/pokeball_icon.png" id="Pokeball" alt="Pokeball">
+		</header><!-- /header -->
 		<!-- depending on how you structure your html, you should have 3 section tags -->
 			<!-- 1) search box -->
+		<main>
+		<section>
+			<input type="search" name="Search Pokemon" value="" placeholder="Search Pokemon">
+		</section>
 			<!-- 2) selected pokemon: picture, property and description of pikachu -->
+		<section>
+			<img id="Pikachu" src="assets/images/pikachu.png" alt="Pikachu">
+			<div id= "infobox">
+				<h2>No. 001 Pikachu </h2>
+				<ul>
+					<li>Electric Type</li>
+					<li>Gentle temperament</li>
+					<li>Grasslands</li>
+				</ul>
+			</div>
+
+			<article id = "description">
+				<p>
+					<span>Pikachu</span> can be seen napping in a bright sunlight. It has two red dots on its cheeks. By practicing html and css <span>Pikachu</span> can learn confusion. <span>Pikachu</span> likes to steal computer chargers from sleeping college students.
+				</p>
+			</<article>
+
+		</section>
 			<!-- 3) other pokemon pictures in 2x6 format -->
+		<section>
 
-		<!-- include navigation buttons -->
+			<div class="grid">
+				<img src="assets/images/whatisthat.png" alt="What is that Pokemon">
+				<img src="assets/images/abra.png" alt="Abra Pokemon">
+				<img src="assets/images/vulpix.png" alt="Vulpix Pokemon">
+				<img src="assets/images/nidorino.png" alt="Nidorino Pokemon">
+				<img src="assets/images/clafary.png" alt="Clafary Pokemon">
+				<img src="assets/images/charmeleon.png" alt="Charmeleon Pokemon">
+				<img src="assets/images/charmander.png" alt="Charmander Pokemon">
+				<img src="assets/images/firething.png" alt="Firething Pokemon">
+				<img src="assets/images/jigglypuff.png" alt="Jigglypuff Pokemon">
+				<img src="assets/images/horsea.png" alt="Horsea Pokemon">
+				<img src="assets/images/pidgey.png" alt="Pidgey Pokemon">
+				<img src="assets/images/wartortle.png" alt="Warturtle Pokemon">
+			</div>
+		</section>
+		</main>
 
+		<!-- include navigation buttons -->
+		<footer>
+			<nav>
+				<ol>
+					<li><button><a href="#Pikachu" title="BACK">BACK</a></button></li>
+					<li><button><a href="map.html" title="MAP">MAP</a></button></li>
+					<li><button><a href="area.html" title="AREA">AREA</a></button></li>
```

Comment:
Extra Credit works good. One interesting thing to note - this will work in chrome and safari but not firefox. Why do you think so ? can you do anything to make it cross paltform